Bruin Bite/Network/Moya.swift

```diff
@@ -106,12 +106,17 @@ extension MainAPI: TargetType {
             return .requestParameters(parameters: ["email": email], encoding: JSONEncoding.default)
         case .matchUser(let user, let meal_times, let meal_day, let meal_period, let dining_halls):
             return .requestParameters(parameters: ["user": user, "meal_times": meal_times, "meal_day": meal_day, "meal_period": meal_period, "dining_halls": dining_halls], encoding: JSONEncoding.default)
+<<<<<<< HEAD
```

Comment:
@sjl1826 you’ve left merge conflicts in here - please fix.

Reply:
woops my bad!